### ISSUE TITLE
Windows Enumeration method causes VP to push Gfx to incorrect monitor/display

### ISSUE THREAD

**Issue by Ltek** [vpinball/vpinball#855]
**Describe the bug**
If Display enumeration is not exactly the same every time, VP will change the display order in its own settings, and the screens will be on the wrong  display... e.g playfield/table gfx will be on the backglass.

This problem is posted in forums and Facebook groups almost weekly this past year I've been using VP. 

**To Reproduce**
Boot your PC and turn on the monitors in a different order than when you set them VP, it will hose VPs settings.

**Expected behavior**
VP display settings should not be dependent on Windows load / enumeration order. VP should "lock" to the physical device object that Windows's PnP manager allocates or the Hardware ID as seen in device manager (like MONITOR\DELD105)

With this is fixed, boot / enumeration order, unplugging / re-plugging, etc will never cause problems. We'll never start a table with gfx on the wrong display

**Versions & Settings**
 - VPX all versions at least since 1.7 (as long as I've been using it)
 - Win 11

**Comment by passionforpins** [comment]
Man I hope this is something that can be done. If you need votes you have mine

**Comment by KrakenByteKraken** [comment]
Good idea!

I vote for this, of course: It could be a dream come true to all of us who must to fight with a few monitors and we're doomed to reset the PC with all the "right displays" before to play VPX.

Please, dear developers: do it!

**Comment by Ltek** [comment]
@toxieainc I suspect hundreds, or thousands, would appreciate if you can fix this in 10.8.1?

**Comment by vbousquet** [comment]
This should be solved in latest nightly builds. Could you confirm this is ok for you ?

**Comment by Ltek** [comment]
> This should be solved in latest nightly builds. Could you confirm this is ok for you ?

Great that you are working on this!  I have been running 10.8 gold code; not had time to test out 10.8.1 builds yet.

**Comment by KrakenByteKraken** [comment]
I usually start my PC with the 4th monitor to work, then, after a hard day working, I switch off it and turn on the 1st, 2nd and 3rd monitors to play pinball.

I noticed that 10.8.1, probably from a few months ago, works fine and start all 3 monitors as it must; but with 10.8 doesn't work exactly the same: I must re-select the playfield monitor or, in other way, restart the PC to play right in 10.8.

In short: 10.8.1 works fine and 10.8 doesn't.

Anyway, thank you for working on this issue, it's near of perfection from day to day.

**Comment by vbousquet** [comment]
Thanks for the feedback. I will close the issue then
